packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -636,10 +637,17 @@ export const Webchat = forwardRef((props, ref) => {
   )
   const webchatReplies = () => <WebchatReplies replies={webchatState.replies} />
 
-  const userInputEnabled = getThemeProperty(
-    'userInput.enable',
-    props.enableUserInput !== undefined ? props.enableUserInput : true
-  )
+  const isUserInputEnabled = () => {
+    let isUserInputEnabled = getThemeProperty(
+      'userInput.enable',
+      props.enableUserInput !== undefined ? props.enableUserInput : true
+    )
+    isUserInputEnabled =
```

Comment:
early return.
WHat do you think about..
```
if (!isUserInputEnabled) {
  return false
}
return !webchatState.isCoverComponentOpen
```

Reply:
I agree in both situations, now it's easier to read 👍